app/workers/migrate_application_choices_worker.rb

```diff
@@ -0,0 +1,37 @@
+class MigrateApplicationChoicesWorker
+  include Sidekiq::Worker
+
+  sidekiq_options queue: :low_priority
+
+  def perform(choice_ids)
+    errors = []
+
+    ApplicationChoice.find(choice_ids).each do |choice|
```

Comment:
should we use find_each or find_in_batches? 
```suggestion
    ApplicationChoice.find(choice_ids).find_each do |choice|
```
```suggestion
    ApplicationChoice.find(choice_ids).each do |choice|
```
```suggestion
    ApplicationChoice.find(choice_ids).each do |choice|
```
```suggestion
    ApplicationChoice.find(choice_ids).each do |choice|
```

Reply:
Yeah this reduced the ram of the sidekiq worker considerably, thanks 👍🏻